tests/test_tasks.py

```diff
@@ -135,17 +136,48 @@ def test_documents_and_requests(task_name: str, task_class: Task):
 
 
 def test_arg_string_task_creation():
-    arg_string = "text_target_separator=\n\n,example_separator=\t,save_examples=False"
+    TEST_EXAMPLE_SEP = "\n===TEST_SEPARATOR===\n"
```

Comment:
These are great, could we have a few such test cases for different separators that could potentially cause issues with the parsing logic?

Reply:
I've added a few more! Let me know if you have any other cases you'd like me to add.

Reply:
how about separators surrounded by square and angle brackets?

Reply:
Oh, why are only whitespace separators supported? Aren't we just concatenating the strings?

Reply:
At the time, I found it out-of-scope for the harness as many templates in `promptsource` already contain a delimiter between context and targets, e.g. the `A:` in  

> "Q: Where is the Eiffel Tower located? A:"

I'm willing to add this in another PR though!